README.md

```diff
@@ -10,6 +19,85 @@ To build this snap locally you need snapcraft. The project must be built as real
 $ sudo snapcraft
 ```
 
+# Testing with spread
+
+## Prerequisites for testing
+
+You need to have the following software installed before you can test with spread
+ - Go (https://golang.org/doc/install or ```sudo snap install go```)
+ - Spread (install from source as per below)
+
+## Installing spread
+
+You can install spread by simply using ```snap install spread```, however this does not allow for the lxd-backend to be used.
+To use the lxd backend you need to install spread from source, as the LXD profile support has not been upstreamed yet.
+This document will be updated with the upstream version when this happens. To install spread from source you need to do the following.
+
+```
+git clone https://github.com/Meulengracht/spread
+cd spread
+go mod init
```

Comment:
I was sort of blindly following these instructions and run into this error:
```
$ go mod init
go mod init: go.mod already exists
```

Reply:
Yea I believe this is my lack of knowledge of how go modules work, I believe this step is not neccessary.